### PR TITLE
UML-615 Improve test coverage - ViewerCodeService

### PR DESCRIPTION
**Pull request by cooperaj**
## Purpose
Improved test coverage

Fixes UML-615

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-api/app/test/AppTest/Service/ViewerCodes/ViewerCodeServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+        $lpaServiceProphecy = $this->prophesize(LpaService::class);
+
+        $service = new ViewerCodeService(
+            $viewerCodeRepoProphecy->reveal(),
+            $userActorLpaRepoProphecy->reveal(),
+            $lpaServiceProphecy->reveal()
+        );
+
+        $service->cancelCode('user_actor_lpa_token', 'user_id', '123412341234');
```
Comment (GemTay) [inline service-api/app/test/AppTest/Service/ViewerCodes/ViewerCodeServiceTest.php:287]: could you assert that $viewerCodeRepoProphecy->cancel() is not called?

Comment (cooperaj) [reply service-api/app/test/AppTest/Service/ViewerCodes/ViewerCodeServiceTest.php:290]: Good shout. Done

File: service-api/app/test/AppTest/Service/ViewerCodes/ViewerCodeServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+    }
+
+    /** @test */
+    public function it_can_cancel_a_code()
+    {
+        $viewerCodeRepoProphecy = $this->prophesize(ViewerCodesInterface::class);
+        $viewerCodeRepoProphecy
+            ->get('123412341234')
+            ->shouldBeCalled()
+            ->willReturn([]);
```
Comment (GemTay) [inline service-api/app/test/AppTest/Service/ViewerCodes/ViewerCodeServiceTest.php:234]: looks like ->get() should return the viewer code data

Comment (cooperaj) [reply service-api/app/test/AppTest/Service/ViewerCodes/ViewerCodeServiceTest.php:234]: Yeah but the code doesn't actually do anything with the data, it just checks that something comes back

File: service-api/app/test/AppTest/Service/ViewerCodes/ViewerCodeServiceTest.php [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+    }
+
+    /** @test */
+    public function it_will_retrieve_codes_of_a_user()
+    {
+        $viewerCodeRepoProphecy = $this->prophesize(ViewerCodesInterface::class);
+        $viewerCodeRepoProphecy
+            ->getCodesByUserLpaActorId('700000000047', 'user_actor_lpa_token')
+            ->shouldBeCalled()
+            ->willReturn([]);
```
[the remark this answers is not in the document]
Comment (cooperaj) [reply service-api/app/test/AppTest/Service/ViewerCodes/ViewerCodeServiceTest.php]: Added something to the return and verified it's passed out of the function.

**Pull request merged**